config/jobs/kubernetes/wg-k8s-infra/trusted/wg-k8s-infra-apps.sh

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
+# Copyright 2021 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# generates wg-k8s-infra app deployment job configs
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SCRIPT_DIR=$(dirname "${BASH_SOURCE[0]}")
+
+readonly OUTPUT="${SCRIPT_DIR}/wg-k8s-infra-apps.yaml"
+# list of subdirs in kubernetes/k8s.io/apps
+readonly APPS=(
+    gcsweb
+    k8s-io
+    kubernetes-external-secrets
+    node-perf-dash
+    perfdash
+    prow
+    publishing-bot
+    sippy
+    slack-infra
+    triageparty-release
+)
+
+cat >"${OUTPUT}" <<EOF
+# DO NOT EDIT. Automatically generated by $0
+
+postsubmits:
+  kubernetes/k8s.io:
+EOF
+
+for app in "${APPS[@]}"; do
+    cat >>"${OUTPUT}" <<EOF
+    - name: post-k8sio-deploy-app-${app}
+      cluster: k8s-infra-prow-build-trusted
+      decorate: true
+      max_concurrency: 1
+      run_if_changed: '^apps\/${app}\/'
+      branches:
+        - ^main$
+      annotations:
+        testgrid-create-test-group: 'true'
+        testgrid-dashboards: wg-k8s-infra-apps
+        testgrid-description: 'runs https://git.k8s.io/k8s.io/apps/${app}/deploy.sh if files change in kubernetes/k8s.io/apps/${app}'
+        testgrid-alert-email: k8s-infra-alerts@kubernetes.io
```

Comment:
```suggestion
        testgrid-alert-email: k8s-infra-rbac-${app}k8s-infra-alerts@kubernetes.io, k8s-infra-alerts@kubernetes.io
```

IMO, workloads only should be informed.

Reply:
So I agree with alerting the app-specific rbac groups (hazily recall trying something like this for staging groups and not having a lot of luck, I'll see what I can find)

Can you clarify what you mean by this statement though?  Do you mean don't add k8s-infra-alerts? Or more than 1 failure before alerting? etc.

One of my next steps (followup PR) was going to be to have these jobs start posting to #k8s-infra-alerts so that we get a better sense of when deploys are working vs. not.

Reply:
Sorry for not be clear. I meant not to add k8s-infra-alerts. Failures alerts should be directed to specific app-specific rbac groups.

> One of my next steps (followup PR) was going to be to have these jobs start posting to #k8s-infra-alerts so that we get a better sense of when deploys are working vs. not.

I'm definitively in favor of this.

Reply:
OK I will remove k8s-infra-alerts as a followup, once I've confirmed these all work.